SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/OutgoingMessages.h

```diff
@@ -0,0 +1,156 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+#pragma once
+
+#include <WorkerSDK/improbable/c_worker.h>
+
+enum class EOutgoingMessageType : int32
+{
+	ReserveEntityIdsRequest,
+	CreateEntityRequest,
+	DeleteEntityRequest,
+	ComponentUpdate,
+	CommandRequest,
+	CommandResponse,
+	CommandFailure,
+	LogMessage,
+	ComponentInterest,
+	EntityQueryRequest,
+	Metrics
+};
+
+struct FOutgoingMessage
```

Comment:
It looks like this enum isn't used. And the struct wrapping it doesn't do anything. So we can remove these and remove it from all the constructors.

Reply:
Good point

Reply:
Welp you can't do it in a nice way in an initializer list so I'm going to keep it.

Reply:
ah wait, I didn't see that we are inheriting from this. Fine.